R/moma-R6.R

```diff
@@ -0,0 +1,109 @@
+#' R6 objects for storing and accessing the results of SFPCA / SFLDA /
+#' SFCCA
+#'
+#' During initialization of an \code{SFPCA} object, \code{R}
+#' calls the \code{C++}-side function, \code{cpp_multirank_BIC_grid_search}, and
+#' wraps the results returned. The \code{SFPCA} object also records penalty levels
+#' and selection schemes of tuning parameters. Several helper
+#' methods are provivded to facilitate access to results.
+#' Initialization is delegated to \code{\link{moma_sfpca}}.
+#'
+#' The following table lists the supported methods for
+#' R6 objects generated by \code{moma_*pca}, \code{moma_*cca}
+#' and \code{moma_*lda} types functions.
+#' \tabular{lccccccc}{
+#'    \tab \code{get_mat_by_index} \tab \code{interpolate}
+#'  \tab \code{print} \tab \code{plot} \tab \code{X_project} \tab \code{Y_project} \tab \code{left_project}
+#'  \cr
+#'  \code{moma_*pca} \tab Yes \tab Yes
+#'   \tab Yes \tab Yes \tab  \tab \tab  Yes
+#'  \cr
+#'   \code{moma_*cca} \tab Yes \tab
+#'  \tab Yes \tab Yes \tab Yes  \tab Yes \tab
+#'  \cr
+#'   \code{moma_*lda} \tab Yes \tab
+#'  \tab Yes \tab Yes \tab Yes \tab  \tab
+#'  \cr
+#' }
+#'
+#' @seealso \code{\link{moma_sfpca}},
+#' \code{\link{moma_sflda}},
+#' \code{\link{moma_sfcca}}
+#'
+#' @section Members:
+#'
+#' \describe{
+#'   \item{
```

Comment:
See above - this might be easier with Markdown syntax.

Reply:
Same.